src/logger/src/lib.rs

```diff
@@ -30,6 +26,11 @@ pub type FcLineWriter = std::io::LineWriter<std::fs::File>;
 /// that are not generally available.
 const DEV_PREVIEW_LOG_PREFIX: &str = "[DevPreview]";
 
+/// The default instance ID.
+pub const DEFAULT_INSTANCE_ID: &str = "anonymous-instance";
+/// The instance ID to use when initializing the logger.
+pub static INSTANCE_ID: OnceLock<String> = OnceLock::new();
```

Comment:
Another, temporary change that I spotted:

You add the `INSTANCE_ID: OnceLock<String>` type in the first commit and then remove it all together.

Reply:
I've squashed d4b7864be15ce89a82a5e700e9afdf8b1307d275 and b1ccbcc21e918e4fcad4e5bd22f0ab295b60460a. I think this should resolve these issues.

Reply:
I liked them separate :( I just wanted to have each commit do exactly what it claims to do.